### PR TITLE
Migrate to GitHub Actions

### PR DESCRIPTION
**Pull request by felipecrs**

### CODE REVIEW

**Comment by felipecrs** [comment]
@joaomoreno the NPM_TOKEN secret must be added to the repository's secrets.

**Comment by joaomoreno** [comment]
Cool! Let's remove the following from this PR:

- removal of concurrency (I can do that)
- updating package-lock.json, I'm not interested in this for now, since it causes more issues than fixes

**Comment by felipecrs** [comment]
@joaomoreno Done

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
+name: ci
+
+on:
+  push:
+    branches: [main]
```
Comment (joaomoreno) [inline .github/workflows/ci.yaml:5]:
```suggestion
    branches: [**]
```

File: .github/workflows/ci.yaml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    if: startsWith(github.ref, 'refs/tags/v')
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js 14
+        uses: actions/setup-node@v2
+        with:
+          node-version: 14
+          registry-url: 'https://registry.npmjs.org'
+      - run: npm install --global npm@latest
```
Comment (joaomoreno) [inline .github/workflows/ci.yaml:39]:
```suggestion
```

**Review by joaomoreno — APPROVED** [review]

**Pull request merged**